server/src/test/java/org/apache/calcite/test/ServerTest.java

```diff
@@ -127,6 +127,16 @@ static Connection connect() throws SQLException {
         assertThat(r.getInt(1), is(1));
         assertThat(r.next(), is(false));
       }
+
+      b = s.execute("create schema if not exists s");
+      assertThat(b, is(false));
+      x = s.executeUpdate("insert into s.t values 2");
+      assertThat(x, is(1));
+
+      b = s.execute("create or replace schema s");
+      assertThat(b, is(false));
```

Comment:
Should this be true as schema s can be replaced?

Reply:
`false` means that the result is not a `ResultSet`